server/src/main/java/org/apache/iotdb/db/engine/memtable/AbstractMemTable.java

```diff
@@ -496,6 +501,11 @@ void updatePlanIndexes(long index) {
     minPlanIndex = Math.min(index, minPlanIndex);
   }
 
+  @Override
+  public int getMemTableId() {
+    return memTableId;
```

Comment:
memTableId  is a fixed value,  memTableIdCounter.getAndIncrement() == 0.

Reply:
Each memtable has its own id, this value is initialized at [AbstractMemTable L87-L88](https://github.com/apache/iotdb/pull/5320/files#diff-9b46aad29da3fa0719055cf485e20ed8c654b8faa799fc03c32ccd9df63f8509R87-R88)